res/country_metadata/popularReligions.yml

```diff
@@ -61,6 +61,7 @@ QA: [muslim]
 SA: [muslim]
 SD: [muslim]
 SG: [buddhist, christian, muslim, chinese_folk, taoist, hindu]
+SI: [christian, muslim]
```

Comment:
This is not really necessary by the way, because Christian and then Muslim is already at the top of the list.